### ISSUE TITLE
Disable skipped run to be rerun in WebApp and some tidy up on messaging

### ISSUE THREAD

**Issue by Anders-Markvardsen** [ISISScientificComputing/autoreduce#844]
Issue raised by: [developer and with users Rob and Duc Le observing]

### What?
In the WebApp we display skipped run, like this GEM run, with is an RB0 run,

![image](https://user-images.githubusercontent.com/1135476/96586993-a7f53400-12d9-11eb-90ff-e7e340b871b2.png)

When a user go into page and rerun results in the error:

```
Request Method: | POST
-- | --
http://reducedev2.isis.cclrc.ac.uk:8080/runs/GEM/confirmation/
3.0.6
TypeError
catching classes that do not inherit from BaseException is not allowed
C:\autoreduction\WebApp\autoreduce_webapp\reduction_viewer\utils.py in createRetryRun, line 189
C:\Apache24\bin\httpd.exe
3.7.4
['C:\\autoreduction\\WebApp\\autoreduce_webapp',  'c:\\python37\\lib\\site-packages\\git\\ext\\gitdb',  'C:\\autoreduction\\WebApp\\autoreduce_webapp',  'C:\\Python37\\python37.zip',  'c:\\python37\\DLLs',  'c:\\python37\\lib',  'C:\\Apache24\\bin',  'c:\\python37',  'c:\\python37\\lib\\site-packages',  'c:\\autoreduction',  'c:\\python37\\lib\\site-packages\\win32',  'c:\\python37\\lib\\site-packages\\win32\\lib',  'c:\\python37\\lib\\site-packages\\Pythonwin',  'c:\\python37\\lib\\site-packages\\IPython\\extensions',  'c:\\python37\\lib\\site-packages\\gitdb\\ext\\smmap',  'c:/autoreduction\\WebApp\\autoreduce_webapp',  'c:/autoreduction\\WebApp\\autoreduce_webapp',  'c:/autoreduction\\WebApp\\autoreduce_webapp',  'C:\\autoreduction\\WebApp\\autoreduce_webapp',  'C:\\autoreduction\\WebApp\\autoreduce_webapp']
```

In this issue fix above error by disable "Re-run with new variables" button.

Furthermore, with this issue tidy a few things with page for skipped run please. For the above run this is thi page:

![image](https://user-images.githubusercontent.com/1135476/96588120-243c4700-12db-11eb-85b7-2f107fd10545.png)

* The top line is really nice "[20 Oct 2020, 1:19 p.m.] Found non-integer RB number: CAL_GEM_04/09/2020 11:54:04" which attempt to explain why a run was skipped. However, it this case clearly wrong message with "non-integer RB number". What it should say in this case is something like "non facility user RB number detected, which follows the format: RBxxxxxxx where xxxxxxx is the 7-digit RB number"
* Change "*Start*: Not yet started" to something like "*Start*: Not run" or same date as Finish field or other
* Change "*Duration*: Not yet finished" to something like "*Duration*: Not applicable" or "*Duration*: Not run" or other

**Comment by keiranjprice101** [comment]
There is a separate issue here where line 189 in `WebApp/autoreduce_webapp/reduction_viewer/utils.py` attempts to catch a module as if it were an exception

**Comment by Anders-Markvardsen** [comment]
Keiran just spotted that RB message picked up by script is CAL_GEM_.... and which is separate from RB0 submitted to ActiveMQ. Not sure what that means

**Comment by keiranjprice101** [comment]
Closed and opened up #855 #853 #856
